src/app/_guards/auth.guard.ts

```diff
@@ -18,15 +18,17 @@ export class AuthGuard implements CanActivate {
                 return true;
             } else {
                 this.router.navigate(['last-7-day-post-list']);
-                window.alert('You are not authorized to access this area!');
+                // window.alert('You are not authorized to access this area!');
```

Comment:
I think we should remove them altogether instead of commenting them out unless you're using them for debugging or something? Up to you